src/server/editorServices.ts

```diff
@@ -477,6 +477,7 @@ export function convertScriptKindName(scriptKindName: protocol.ScriptKindName) {
 
 /** @internal */
 export function convertUserPreferences(preferences: protocol.UserPreferences): UserPreferences {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { lazyConfiguredProjectsFromExternalProject, ...userPreferences } = preferences;

```

Comment:
Do we really need to filter out `lazyConfiguredProjectsFromExternalProject` option. Can we instead pass it as is ?

Reply:
I have no idea; I should probably go `git blame` this back to where it was added to see if there was a reason this was written this way.

Reply:
Traces back to https://github.com/microsoft/TypeScript/commit/03bb5d170ec in #26716.

Reply:
i think it should be ok to pass it as is and avoid having to destructure object all together when its not needed.

Reply:
Yeah, I'll give it a shot, though there are other fundamental problems with this PR that probably mean we're not going to do this, e.g. the 14s extra lint time because ts-eslint doesn't support jsdoc itself, and the only available parser is very slow (for some reason... but TS already parses things, so I don't understand what's happening).